### PR TITLE
Minor issues

### PR DESCRIPTION
**Pull request by taraganepola**
## Checklist

- [x] Changes have been tested

### CODE REVIEW

File: dashboard/dashboard/pages/patients/__init__.py [hunk 1]
```diff
@@ -324,8 +324,11 @@ def biground(x, base=5):
         )
         
 
-    fig.update_layout(barmode="overlay")
-    fig.update_traces(opacity=0.75)
+    fig.update_layout(barmode="overlay", bargap=0.1)
+    fig.update_traces(
+        opacity=0.75, 
+        hovertemplate='%{x}: %{y:.2f}<extra></extra>',
```
Comment (imrehg) [inline dashboard/dashboard/pages/patients/__init__.py:330]: What does that `<extra></extra>` part mean?

File: dashboard/dashboard/pages/quality/columns.py [hunk 2]
```diff
@@ -125,7 +125,7 @@
 DATE_COLUMNS = [
     # "filename_earliest_date",
     # "filename_latest_date",
-    "swab_date",
+    # "swab_date", # only in negative patients
```
Comment (imrehg) [inline dashboard/dashboard/pages/quality/columns.py:128]: If commented out, I think can remove the whole line.

File: dashboard/dashboard/pages/patients/__init__.py (hunk 1 above)
Comment (taraganepola) [reply dashboard/dashboard/pages/patients/__init__.py:330]: without this you get an extra hover annotation which tells you the trace name e.g., "trace0". I didn't like how it looked so removed.
https://plotly.com/python/hover-text-and-formatting/#customizing-hover-text-with-a-hovertemplate

**Review by imrehg — APPROVED** [review]
LGTM, please wait till the morning to merge, just in case :)

**Pull request merged**